views/homepage.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1> This is the homepage of Mick Jagger, welcome people!!</h1>
+    <nav>
+        <a href ="/views/about.html"> About</a> 
```

Comment:
The `href` attribute should have the endpoint you want to call in the backend as its value, not the path to the file in your computer! According to your app.js (line 10), its value should be '/about'. ✌🏻